### ISSUE TITLE
in-label-space设置下conll测试集问题

### ISSUE THREAD

**Issue by qhq123** [AndrewHYC/COPNER#5]
您好！ 在task=in-label-space，mode=conll设置下，我将framework.py中的517-521行：logits, pred, _ = model(query['inputs'], query['tagging_labels'], query['index_labels'],query['target_classes'],query['sentence_num'])更改为了logits, pred, _ = model(query['inputs'], query['batch_labels'],query['valid_masks'],query['target_classes'],query['sentence_num'])使它可以运行，但是在eval阶段读取test.txt时好像会出现读取单行数据（长度过短）时出现异常。
是我不应该这么修改吗？（但是不修改的话代码无法运行，因为代码中没有定义过query中的字段）还是其他原因？

**Comment by mihua5329** [comment]
你好 可以求一下数据集或者处理脚本吗

**Comment by mihua5329** [comment]
> 您好！ 在task=in-label-space，mode=conll设置下，我将framework.py中的517-521行：logits, pred, _ = model(query['inputs'], query['tagging_labels'], query['index_labels'],query['target_classes'],query['sentence_num'])更改为了logits, pred, _ = model(query['inputs'], query['batch_labels'],query['valid_masks'],query['target_classes'],query['sentence_num'])使它可以运行，但是在eval阶段读取test.txt时好像会出现读取单行数据（长度过短）时出现异常。 是我不应该这么修改吗？（但是不修改的话代码无法运行，因为代码中没有定义过query中的字段）还是其他原因？

你好 可以求一下数据集或者处理脚本吗 我的邮箱 2350321824@qq.com 感谢！

**Comment by bule-code** [comment]
你好 可以求一下数据集吗 我的邮箱289016186@qq.com感谢！可商量

**Comment by bule-code** [comment]
> 您好！ 在task=in-label-space，mode=conll设置下，我将 framework.py 中的517-521行：logits， pred， _ = model（query['inputs']， query['tagging_labels']， query['index_labels']，query['target_classes']，query['sentence_num']）更改为了logits， pred， _ = model（query['inputs']， query['batch_labels']，query['valid_masks']，query['target_classes']，query['sentence_num']）使它可以运行，但是在eval阶段读取test.txt时好像会出现读取单行数据（长度过短）时出现异常。 （但是不修改的话代码无法运行，因为代码中没有定义过query中的字段）还是其他原因？

你好 可以求一下数据集吗 我的邮箱[289016186@qq.com](mailto:289016186@qq.com)感谢！可商量
